src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -378,6 +381,27 @@ public ReadBalanceBehavior ReadBalanceBehavior
             }
         }
 
+        public int WriteBalanceSeed
+        {
+            get => _writeBalanceSeed;
+            set
+            {
+                AssertNotFrozen();
+                _writeBalanceSeed = value;
+            }
+        }
+
+        public WriteBalanceBehavior WriteBalanceBehavior
```

Comment:
From implementation it more looks like `LoadBalanceBehavior` than `WriteBalanceBehavior`. I think we should switch the name.